contrib/builder/deb/armhf/raspbian-jessie/Dockerfile

```diff
@@ -6,8 +6,8 @@ RUN sed -i s/httpredir.debian.org/$APT_MIRROR/g /etc/apt/sources.list
 
 RUN apt-get update && apt-get install -y apparmor bash-completion btrfs-tools build-essential curl ca-certificates debhelper dh-apparmor dh-systemd git libapparmor-dev libdevmapper-dev libltdl-dev libsqlite3-dev libsystemd-journal-dev --no-install-recommends && rm -rf /var/lib/apt/lists/*
 
-ENV GO_VERSION 1.6.3
-ENV GOARM 6
+ENV GO_VERSION 1.7
+ENV GOARM 7
```

Comment:
@runcom no this should be unchanged, it is the Arm architecture version, unrelated to Go version.